forge-core/src/main/java/forge/ImageKeys.java

```diff
@@ -367,26 +370,29 @@ public static File setLookUpFile(String filename, String fullborderFile) {
         return null;
     }
     private static File findFile(String dir, String filename) {
-        if (dir.equals(CACHE_CARD_PICS_DIR)) {
-            for (String ext : FILE_EXTENSIONS) {
-                if (ext.equals(""))
+        for (String ext : FILE_EXTENSIONS) {
```

Comment:
I think I'd rather just leave this function alone and convert filename to normalized at the top, and don't try to rename. I'm not sure how writing file names in different character sets is going to react, and it feels like the type of thing thats just going to cause more problems than its solving.

Reply:
i need to check this out, these characters should all be supported by the different file systems

if i would normalize the filename at the top, it would fail to find the already existing Filenames

@paulsnoops didn't give me the Info i would need how to work with old existing Filenames

Reply:
Not sure why you think that's his responsibility. He was just pointing out sobering he thought of. He's not really the expert of that area

Reply:
He said the old cache names will be a problem

Otherwise I would revert the changes to this function, and have it be done at later MR like I said multiple times